app/models/identity.rb

```diff
@@ -14,50 +13,9 @@ class Identity < OmniAuth::Identity::Models::ActiveRecord
   validates :user, presence: true
   validates :user_id, uniqueness: true
 
-  # Support for legacy CNX passwords
-  def authenticate(unencrypted_password)
-    if is_password_digest_ssha?
-      authenticate_with_ssha unencrypted_password
-    else
-      super
-    end
-  end
-
   # Returns true if the user is due for resetting their password
   def password_expired?
+    return if DEFAULT_PASSWORD_EXPIRATION_PERIOD.nil?
```

Comment:
I could see us wanting someone to reset their password even if we currently don't make passwords expired in general

Reply:
So just remove this return statement?

Reply:
Yes but check that we don't have a bunch of expired people in the DB already or something :)

Reply:
![Screen Shot 2022-09-07 at 13 25 28](https://user-images.githubusercontent.com/3905516/188951295-d30080dc-cc4b-4545-bc19-4430eb7987dc.png)
Looks like we have about 35k :/

Reply:
![Screen Shot 2022-09-07 at 13 26 28](https://user-images.githubusercontent.com/3905516/188951479-c217a655-34b3-40d0-9a29-e14551781226.png)
Ah - but we only care if they are **going** to expire, yeah?